src/validation.cpp

```diff
@@ -3435,6 +3435,9 @@ bool ProcessNewBlockHeaders(const std::vector<CBlockHeader>& headers, CValidatio
         }
     }
     NotifyHeaderTip();
+    if (IsInitialBlockDownload() && *ppindex && pindexBestHeader) {
```

Comment:
Isn't cs_main required for pindexBestHeader?

Reply:
Hhhmp.. fixed.